### PR TITLE
facebook updated to latest SDK and moved unwamted dependencie to gradle

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: android/FacebookPlugin.java [hunk 1]
```diff
@@ -951,7 +951,7 @@ public void onError(FacebookException exception) {
       requestDialog.registerCallback(callbackManager, new FacebookCallback<GameRequestDialog.Result>() {
           public void onSuccess(GameRequestDialog.Result result) {
               log("{facebook} game request result - success");
-              sendResponse(result.getRequestData(), null, activeRequest);
+              sendResponse(result.getRequestRecipients(), null, activeRequest);
```
Comment (vivekb91) [inline android/FacebookPlugin.java]: this change is verifying

File: android/gradletealeaf.xml [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-    implementation 'com.facebook.android:facebook-messenger:4.34.0'
-    implementation 'com.facebook.android:facebook-applinks:4.34.0'
-    implementation 'com.facebook.android:facebook-marketing:4.34.0'
+    implementation 'com.facebook.android:facebook-core:4.41.0'
+    implementation 'com.facebook.android:facebook-login:4.41.0'
+    implementation 'com.facebook.android:facebook-places:4.41.0'
+    implementation 'com.facebook.android:facebook-messenger:4.41.0'
+    implementation 'com.facebook.android:facebook-applinks:4.41.0'
+    implementation 'com.facebook.android:facebook-marketing:4.41.0'
+    implementation 'com.facebook.android:facebook-share:4.41.0'
```
Comment (rampr) [inline android/gradletealeaf.xml:5]: will we need all of these @vivekb91 ?

File: android/config.json [hunk 3]
```diff
@@ -6,5 +6,6 @@
         "FacebookSDK400/facebook/build/outputs/aar/facebook-debug.aar"
```
Comment (rampr) [inline android/config.json]: since you have moved to gradle, is this still needed ?

Comment (vivekb91) [reply android/config.json]: No, I will remove

File: android/gradletealeaf.xml (hunk 2 above)
Comment (vivekb91) [reply android/gradletealeaf.xml:5]: I will check, All these may not be needed

Comment (vivekb91) [reply android/gradletealeaf.xml:5]: Removed unwanted libs

File: android/config.json (hunk 3 above)
Comment (vivekb91) [reply android/config.json]: Removed

**Review by nasimafeef — APPROVED** [review]

**Review by rampr — APPROVED** [review]

**Pull request merged**